### PR TITLE
[IOPID-840] : Adding Mixpanel Track Event Logout-Flow

### PR DESCRIPTION
**Pull request by claudiodall**
## Short Description
Adding Mixpanel Track Event for the **Logout-Flow.**

## How to Test
Run the project and check the console for the tracked event (mixpanel mocked), as specified in Figma, for the logout flow.

### CODE REVIEW

File: src/app/[locale]/(pages)/esci/errore/page.tsx [hunk 1]
```diff
@@ -21,11 +31,16 @@ const LogOutKo = (): React.ReactElement => {
...
+                trackEvent(
+                  isL1 ? 'IO_SESSION_EXIT_TRY_AGAIN' : 'IO_PROFILE_SESSION_EXIT_TRY_AGAIN'
+                ),
             }}
             secondButton={{
               variant: 'outlined',
-              href: ROUTES.PROFILE,
-              text: t('common.backtoprofile'),
+              href: isL1 ? ROUTES.LOGIN : ROUTES.PROFILE,
+              text: isL1 ? t('common.close') : t('common.backtoprofile'),
```
Comment (shadowsheep1) [inline src/app/[locale]/(pages)/esci/errore/page.tsx:42]: Is this a previous bug or regression?

File: src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx [hunk 2]
```diff
@@ -67,8 +68,18 @@ export function FeedbackMessage({
...
+                      if (firstButton.onClick) {
+                        firstButton.onClick();
+                      }
+                    }
+                  : () => {
+                      pushWithLocale(firstButton.href || '/');
+                      if (firstButton.onClick) {
+                        firstButton.onClick();
+                      }
+                    }
```
Comment (shadowsheep1) [inline src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx:82]: How is this related with this PR?

File: src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx [hunk 3]
```diff
@@ -79,7 +90,12 @@ export function FeedbackMessage({
                 sx={{
                   marginLeft: 2,
                 }}
-                onClick={() => pushWithLocale(secondButton.href || '')}
+                onClick={() => {
+                  pushWithLocale(secondButton.href || '');
+                  if (secondButton.onClick) {
+                    secondButton.onClick();
+                  }
+                }}
```
Comment (shadowsheep1) [inline src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx:98]: as above

File: src/app/[locale]/(pages)/esci/conferma/page.tsx [hunk 4]
```diff
@@ -7,13 +7,22 @@ import { WebProfileApi } from '@/api/webProfileApiClient';
...
 const LogoutConfirm = (): React.ReactElement => {
   const [sessionData, setSessionData] = useState<SessionState>();
   const t = useTranslations('ioesco');
   const userFromStorage = storageUserOps.read();
   const isL1 = userFromStorage?.spidLevel === process.env.NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L1;
 
+  useEffect(() => {
+    if (sessionData) {
+      trackEvent(isL1 ? 'IO_SESSION_EXIT_STATUS_PAGE' : 'IO_PROFILE_SESSION_EXIT_STATUS_PAGE', {
+        session_status: sessionData.session_info.active ? 'on' : 'off',
```
Comment (shadowsheep1) [inline src/app/[locale]/(pages)/esci/conferma/page.tsx:21]: This has been use already twice. Would we consider to put this login inside an utility function?

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: src/app/[locale]/(pages)/esci/errore/page.tsx (hunk 1 above)
Comment (claudiodall) [reply src/app/[locale]/(pages)/esci/errore/page.tsx:42]: doing the task, I realized that it was never corrected

File: src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx (hunk 2 above)
Comment (claudiodall) [reply src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx:82]: Yes, I introduced it to have a callback for those buttons in the component to handle the call back events, as each page is different

File: src/app/[locale]/(pages)/esci/conferma/page.tsx (hunk 4 above)
Comment (claudiodall) [reply src/app/[locale]/(pages)/esci/conferma/page.tsx:21]: Where else has it been used?

File: src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx (hunk 3 above)
Comment (claudiodall) [reply src/app/[locale]/_component/feedbackMessage/feedbackMessage.tsx:98]: Yes, I introduced it to have a callback for those buttons in the component to handle the call back events, as each page has different trackevent

File: src/app/[locale]/(pages)/esci/conferma/page.tsx (hunk 4 above)
Comment (shadowsheep1) [reply src/app/[locale]/(pages)/esci/conferma/page.tsx:21]: [Here](https://github.com/pagopa/io-web-profile/blob/494f73370586a7cdecc36f2b4366bbdf67f5a6c2/src/app/%5Blocale%5D/page.tsx#L29) if I'm not wrong. Check it out...

Comment (claudiodall) [reply src/app/[locale]/(pages)/esci/conferma/page.tsx:21]: you're right, i will create common utility

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]
Looking at the event table on confluence I saw this event too: `IO_PROFILE_SESSION_EXIT_START`. It looks like it's missing. Am I right?

**Comment by claudiodall** [comment]
> Looking at the event table on confluence I saw this event too: `IO_PROFILE_SESSION_EXIT_START`. It looks like it's missing. Am I right?

Yes, just added with last commit

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Pull request merged**